primitives/evm/src/lib.rs

```diff
@@ -129,9 +129,6 @@ impl WeightInfo {
 			(self.ref_time_usage, self.ref_time_limit)
 		{
 			let ref_time_usage = self.try_consume(cost, ref_time_limit, ref_time_usage)?;
-			if ref_time_usage > ref_time_limit {
```

Comment:
If I understand correctly this is dangerous as it allows a an extrinsic to go over the limit, no ?

Reply:
It was a duplicated check, `try_consume` in the line above was already doing it.
https://github.com/moonbeam-foundation/frontier/blob/56349814c1e029d99a9c433cc229e81d32cb0aff/primitives/evm/src/lib.rs#L118-L123